### ISSUE TITLE
test for bagging svm

### ISSUE THREAD

**Issue by tpaquier** [tpaquier/material_science#8]
Doubt but maybe overthinking it : 
Check if the train set is really the train set and during the prediction of the test, there is no 'new training' on the test set as I might have mixed up the datasets. 
--> main problem would be that in this case the training set is completely useless.
